nva/nva.c

```diff
@@ -135,6 +147,84 @@ struct nva_card *nva_init_apu(struct pci_device *dev) {
 	return card;
 }
 
+static int check_modalias(const char *node_name) {
+	char path[PATH_MAX];
+	char buffer[255] = { 0 };
+	FILE *file;
+
+	/* Check if the GPU is indeed made by NVIDIA by checking the compatibility
```

Comment:
There are two problems here:

- the MMIO area might not be 16MB
- runing parse_pmc_id on them won't work